stat/combin/combin_test.go

```diff
@@ -179,3 +182,96 @@ func TestCombinationGenerator(t *testing.T) {
 		}
 	}
 }
+
+func TestCartesian(t *testing.T) {
+	// First, test with a known return.
+	data := [][]float64{
+		{1, 2},
+		{3, 4},
+		{5, 6},
+	}
+	want := mat.NewDense(8, 3, []float64{
+		1, 3, 5,
+		1, 3, 6,
+		1, 4, 5,
+		1, 4, 6,
+		2, 3, 5,
+		2, 3, 6,
+		2, 4, 5,
+		2, 4, 6,
+	})
+	got := Cartesian(nil, data)
+	if !mat.Equal(want, got) {
+		t.Errorf("cartesian data mismatch. \nwant:\n%v\ngot:\n%v", mat.Formatted(want), mat.Formatted(got))
```

Comment:
Here and in the next errorf remove the space before the `\n`.

Reply:
Done.